### DISCUSSION TITLE
Amazing tool!

### DISCUSSION THREAD

**Discussion by sermuns** [mifi/lossless-cut#1454] [General]
I am currently digitising my family's VHS collection and your tool has been absolutely incredible in the process. Thank you sooo so much!

**Comment by mifi** [comment]
glad u like it!

**Comment by vegeta897** [comment]
I just want to add my appreciation here. I've been using this tool since 2019 and it was already awesome back then.

One detail I really like is how the "Play selected segments" button is styled with a gradient of the segment colors.
